test/hashing.test.ts

```diff
@@ -0,0 +1,11 @@
+import { describe, expect, it } from "bun:test";
+import { HASH_CONFIG } from "../src/utility/constants.ts";
+
+describe("Hashing Tests", () => {
+	it("should hash a password", () => {
+		const password = "1234567890";
+		const hashedPassword = Bun.password.hashSync(password, HASH_CONFIG);
```

Comment:
Vervang `hashSync` met `hash` en gebruik `await`.
```suggestion
		const hashedPassword = await Bun.password.hashSync(password, HASH_CONFIG);
```